apps/todoapp/src/app/todos/services/auth.service.ts

```diff
@@ -21,12 +23,12 @@ export class AuthService {
           // User is authenticated, proceed with your application
           console.log('User is authenticated:', user);
           this.user$.next(user as DiscordUser);
-        },
-        (error) => {
-          // 403 response, user is not authenticated
```

Comment:
Unauthenticated response code should be 401 not 403. 403 is for unauthorized access with correct token but forbidden access to some resource.